packages/block-editor/src/components/child-layout-control/index.js

```diff
@@ -104,36 +145,52 @@ export default function ChildLayoutControl( {
 							value={ flexSize }
 						/>
 					) }
-				</>
+				</VStack>
 			) }
 			{ parentLayoutType === 'grid' && (
-				<HStack>
-					<InputControl
-						size={ '__unstable-large' }
+				<HStack style={ { gridColumn: '1 / -1' } }>
```

Comment:
This is a bit dodgy but I couldn't get `spacing={ 2 }` to work with `HStack`. Cc @mirka is there a better way to do it? 😅